### PR TITLE
“#7Task. Dzmitry Bryt”

### PR DESCRIPTION
**Pull request by Daqueze21**
https://travis-ci.org/github/Daqueze21/js-assignments/builds/715472558

### CODE REVIEW

**Comment by FedorovD** [comment]
Please clear PR code not related to Task#7 and click "ready for review" when it will be finished. Thank you!

**Comment by Daqueze21** [comment]
> Please clear PR code not related to Task#7 and click "ready for review" when it will be finished. Thank you!

I need to remove code from Conflicting files:
README.md
package-lock.json ???

**Comment by FedorovD** [comment]
image
Remove these changes from Task1-4, there should only be a task#7

**Comment by Daqueze21** [comment]
Remove solutions from tasks 1-4. My fault!

**Comment by Daqueze21** [comment]
Update files

File: task/07-yield-tasks.js [hunk 1]
```diff
@@ -126,7 +165,22 @@ function* breadthTraversalTree(root) {
  *   [ 1, 3, 5, ... ], [ -1 ] => [ -1, 1, 3, 5, ...]
  */
 function* mergeSortedSequences(source1, source2) {
-    throw new Error('Not implemented');
+    let gen1 = source1();
+    let gen2 = source2();
+    
+    while (true) {
```
Comment (FedorovD) [inline task/07-yield-tasks.js]: sequences can be finite. Add exit condition

**Comment by FedorovD** [comment]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**